pkg/util/metrics/system/cmdline_stats_test.go

```diff
@@ -0,0 +1,29 @@
+package system
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCmdlineStats(t *testing.T) {
```

Comment:
We need to the test the following:
1.) Use COS's default
2.) cmdline with key=value
3) cmdline with key="val val2="
3.) cmdline with single value like "rghb"

Reply:
+1

Reply:
added the testcases